clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts

```diff
@@ -140,10 +140,12 @@ export class CopybookDownloadService {
     dialectType: string,
   ): Promise<string | undefined> {
     if (this.handleAsEndevorElement(documentUri)) {
-      return await this.e4eDownloader?.getE4ECopyBookLocation(
-        copybookName,
-        documentUri,
-      );
+      return (
+        await this.e4eDownloader?.getE4ECopyBookLocation(
+          copybookName,
+          documentUri,
+        )
+      )?.toString();
```

Comment:
I suggest either
```suggestion
      const copybookUri = await this.e4eDownloader?.getE4ECopyBookLocation(
        copybookName,
        documentUri,
      );
      return copybookUri?.toString();
```
or
```suggestion
      return this.e4eDownloader?.getE4ECopyBookLocation(
        copybookName,
        documentUri,
      ).then(uri => uri?.toString());
```